presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/PickLayout.java

```diff
@@ -1,90 +0,0 @@
-/*
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package com.facebook.presto.sql.planner.optimizations;
-
-import com.facebook.presto.Session;
-import com.facebook.presto.metadata.Metadata;
-import com.facebook.presto.spi.type.Type;
-import com.facebook.presto.sql.planner.PlanNodeIdAllocator;
-import com.facebook.presto.sql.planner.Symbol;
-import com.facebook.presto.sql.planner.SymbolAllocator;
-import com.facebook.presto.sql.planner.plan.FilterNode;
-import com.facebook.presto.sql.planner.plan.PlanNode;
-import com.facebook.presto.sql.planner.plan.SimplePlanRewriter;
-import com.facebook.presto.sql.planner.plan.TableScanNode;
-import com.facebook.presto.sql.tree.BooleanLiteral;
-
-import java.util.Map;
-
-import static java.util.Objects.requireNonNull;
-
-/**
- * Pick an arbitrary layout if none has been chosen
- */
-@Deprecated
-public class PickLayout
```

Comment:
Please leave this class for a while

Reply:
but it's not used anymore.
It can't be the non-iterative alternative to PickTableLayout because it only gets run once, so you can't run it too early or even later if there's a table layout that's chosen by something else earlier (e.g. PickTableLayout).